src/main/kotlin/parts/wisdom/arcraider/Arc.kt

```diff
@@ -16,7 +16,7 @@ data class Task(
 @Serializable
 data class ArcPair(
     val input: SerializedGrid,
-    val output: SerializedGrid
+    val output: SerializedGrid = listOf(emptyList())
```

Comment:
Seems unexpected and hence error-prone to initialize `input` and `output` differently.

Reply:
Ahh this is a leftover hack I did because all tasks will have an input, but some will not have an output (the 'test' ones). 
We need to better address that scenario. I'm removing this from this PR and opening an issue.